### PR TITLE
Create 1.18 support tags

### PR DESCRIPTION
**Pull request by Stonewall0210**
This PR adds two tags: one with a short FAQ about 1.18 support, and one with a fix for the somewhat common `Please connect through the official Geyser` error.

I'm hoping to add more 1.18 related FAQs in the future but for now I have compiled two common one message fixes.

### CODE REVIEW

File: src/main/resources/tags/info/1.18support.tag [hunk 1]
```diff
@@ -0,0 +1,10 @@
+type: text
+aliases: 1.18
+
+---
+
+Geyser's latest release supports 1.17.1 through to 1.18.
```
Comment (Chew) [inline src/main/resources/tags/info/1.18support.tag:6]: This is misleading, Geyser supports 1.12 through 1.18 but the bedrock player will be joined as 1.18. The ViaVersion part is right though.

File: src/main/resources/tags/info/floodgatemismatch.tag [hunk 2]
```diff
@@ -0,0 +1,7 @@
+type: text
```
Comment (Camotoy) [inline src/main/resources/tags/info/floodgatemismatch.tag:1]: Might want to mention there that this is not the issue if all Floodgate instances on a particular network are updated to 2.1.0-SNAPSHOT.

File: src/main/resources/tags/info/1.18support.tag (hunk 1 above)
Comment (Camotoy) [inline src/main/resources/tags/info/1.18support.tag:6]: Geyser only supports Java Edition 1.18 and Bedrock Edition 1.17.30 - 1.18 at this time.

Comment (Stonewall0210) [reply src/main/resources/tags/info/1.18support.tag:6]: Oh sorry, I didn't realize that. I'll quickly fix

File: src/main/resources/tags/info/floodgatemismatch.tag (hunk 2 above)
Comment (Stonewall0210) [reply src/main/resources/tags/info/floodgatemismatch.tag:1]: Does this fix work?

**Comment by Camotoy** [comment]
On continued thought, I think the 1.18 tag is something that should already be covered by the existing ViaVersion and the `!!olderversions` tag.

**Comment by Stonewall0210** [comment]
> On continued thought, I think the 1.18 tag is something that should already be covered by the existing ViaVersion and the `!!olderversions` tag.

Does this make the 1.18 tag unnecessary?

**Comment by Camotoy** [comment]
I would say so.

**Comment by Camotoy** [comment]
I'm going to close this as the contents of both tags are obsolete at the moment. If you think there is an improvement to be had elsewhere, feel free to let us know here or make a new PR!

**Pull request closed without merging**